src/units.rs

```diff
@@ -0,0 +1,57 @@
+/// This structure represent all the stats that a unit can have.
+struct Stats {
+    /// he distance the model can Advance Move in inches per turn.
+    advance_rate: usize,
+    /// The distance the model can March Move in inches per turn.
+    march_rate: usize,
+    /// Represents the model’s overall morale, bravery and a unit’s ability to quickly reorganise during battle.
+    discipline: usize,
+    /// When the model loses this many Health Points, it is removed as a casualty
+    health_point: isize,
+    /// How well the model avoids being hit in melee. A high Defensive Skill may also represent models with equipment useful for parrying melee attacks, such as shields. 
```

Comment:
```suggestion
    /// How well the model avoids being hit in melee.
```
The last part isn't really useful